### PR TITLE
Added plt and gotplt commands to print the .plt and .got.plt sections, and cleaned up elfheader command

### PR DESCRIPTION
**Pull request by gsingh93**
I'm wondering if we should just expose the `print_symbols_in_section` command, or is exposing these two commands fine (or expose both)?

Example output:
```
pwndbg> plt
0x4007d0: free@plt
0x4007e0: malloc_info@plt
0x4007f0: puts@plt
0x400800: printf@plt
0x400810: mprobe@plt
0x400820: malloc_usable_size@plt
0x400830: __libc_start_main@plt
0x400840: fgets@plt
0x400850: strcmp@plt
0x400860: __gmon_start__@plt
0x400870: mcheck_pedantic@plt
0x400880: malloc@plt
0x400890: __isoc99_sscanf@plt
0x4008a0: malloc_stats@plt
0x4008b0: mcheck@plt
0x4008c0: mcheck_check_all@plt

pwndbg> gotplt
0x602018: free@got.plt
0x602020: malloc_info@got.plt
0x602028: puts@got.plt
0x602030: printf@got.plt
0x602038: mprobe@got.plt
0x602040: malloc_usable_size@got.plt
0x602048: __libc_start_main@got.plt
0x602050: fgets@got.plt
0x602058: strcmp@got.plt
0x602060: __gmon_start__@got.plt
0x602068: mcheck_pedantic@got.plt
0x602070: malloc@got.plt
0x602078: __isoc99_sscanf@got.plt
0x602080: malloc_stats@got.plt
0x602088: mcheck@got.plt
0x602090: mcheck_check_all@got.plt

pwndbg> elfheader
0x400238 - 0x400254 .interp
0x400254 - 0x400274 .note.ABI-tag
0x400274 - 0x400298 .note.gnu.build-id
0x400298 - 0x4002c0 .gnu.hash
0x4002c0 - 0x400488 .dynsym
0x400488 - 0x400572 .dynstr
0x400572 - 0x400598 .gnu.version
0x400598 - 0x4005d8 .gnu.version_r
0x4005d8 - 0x400620 .rela.dyn
0x400620 - 0x4007a0 .rela.plt
0x4007a0 - 0x4007ba .init
0x4007c0 - 0x4008d0 .plt
0x4008d0 - 0x400e02 .text
0x400e04 - 0x400e0d .fini
0x400e10 - 0x401023 .rodata
0x401024 - 0x401068 .eh_frame_hdr
0x401068 - 0x40119c .eh_frame
0x601e10 - 0x601e18 .init_array
0x601e18 - 0x601e20 .fini_array
0x601e20 - 0x601e28 .jcr
0x601e28 - 0x601ff8 .dynamic
0x601ff8 - 0x602000 .got
0x602000 - 0x602098 .got.plt
0x602098 - 0x6020a8 .data
0x6020b0 - 0x6020c8 .bss
```

### CODE REVIEW

File: pwndbg/commands/elf.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+@pwndbg.commands.Command
+def plt():
+    """
+    Prints any symbols found in the .plt section if it exists.
+    """
+    print_symbols_in_section('.plt', '@plt')
+
+def get_section_bounds(section_name):
+    section_name = section_name.encode('ascii')
+    exe = str(pwndbg.auxv.get()['AT_EXECFN'])
```
Comment (zachriggle) [inline pwndbg/commands/elf.py]: `pwndbg.proc.exe` (it works now!)

File: pwndbg/commands/elf.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+def elfheader():
+    """
+    Prints the section mappings contained in the ELF header.
+    """
+    exe = str(pwndbg.auxv.get()['AT_EXECFN'])
+    with open(exe, 'rb') as f:
+        elffile = ELFFile(f)
+        load_segment = elffile.get_segment(3)
+        segment_base = load_segment['p_vaddr']
+        for section in elffile.iter_sections():
```
Comment (zachriggle) [inline pwndbg/commands/elf.py]: Can we print these in linear order? I'm not sure they're guaranteed to be, by default

Comment (gsingh93) [inline pwndbg/commands/elf.py]: The order is the order that the sections are declared in the binary. You're saying you want them linear by starting address? I can do that.

**Comment by gsingh93** [comment]
Rebased and updated.

**Comment by zachriggle** [comment]
Or alphabetic, I'm not sure which is more useful.  Unordered / file order seems strictly worse than either of the other two options.

**Comment by gsingh93** [comment]
I added a `get_file` function to handle downloading the file.

**Pull request merged**